core/src/components/button/button.ios.vars.scss

```diff
@@ -83,7 +83,7 @@ $button-ios-large-height:                             3.1em !default;
 $button-ios-large-border-radius:                      16px !default;
 
 /// @prop - Font size of the large button
-$button-ios-large-font-size:                          20px !default;
+$button-ios-large-font-size:                          min(1.25rem, 60px) !default;
```

Comment:
I used 3x the value of the `font-size` at `100%` to calculate the `max`. In this case `20px * 3`.